front/app/containers/Admin/messaging/CustomEmails/CampaignForm/index.tsx

```diff
@@ -125,27 +123,22 @@ const CampaignForm = ({
     return [
       {
         value: 'author',
-        label: !isNilOrError(user) ? getFullName(user.data) : '',
+        label: getFullName(user.data),
```

Comment:
Out of scope for this PR, but I wonder if we should have a backup here which just uses "user" or "resident" or something for cases where there is no name (E.g. Light users) :thinking:

Reply:
This is the author/sender of the email (the admin in this case). We're in the admin here, so we can assume the user account exists or the page would not be loaded in the first place. Renamed to authUser to make it clearer.

Reply:
Ah, gotchya! :+1: